core/domain/config_domain_test.py

```diff
@@ -91,6 +91,11 @@ def test_config_property_schemas_are_valid(self):
                 property_name).schema
             schema_utils_test.validate_schema(schema)
 
+        schemas = config_domain.Registry.get_config_property_schemas()
+        for property_name in (
+                config_domain.Registry.get_all_config_property_names()):
+            schema_utils_test.validate_schema(schemas[property_name]['schema'])
```

Comment:
(Sorry I'm finding it difficult to understand) How is this test different than the test on line 92? Can we have a separate test block/functions if they are testing two different behaviors?

Reply:
The code from lines 88-92 checks if the method `get_config_property( )` can fetch all the property instances individually while the code from lines 94-97 checks if `get_config_property_schemas( )` fetches all the property instances correctly. 

Since both of these checks deal with fetching and validating config property schemas, I clubbed them together.

Reply:
Thanks fro the explanation, @IamEzio